mysite/announcements/tasks.py

```diff
@@ -23,10 +23,14 @@ def check_scheduled_announcements():
 
         for u in user_list:
             print("{} {} {}".format(u, announcement.title, announcement.message))
-            updated_announcement_ids.append(announcement.id)
+            Status.objects.create(announcement_id=announcement.id, user_id=u.id,
```

Comment:
We are creating these in loop, we can utilize something like [bulk_create](https://docs.djangoproject.com/en/2.0/ref/models/querysets/#bulk-create) here.

Reply:
Bulk_create doc says it doesn't work with many to many relationships

Reply:
It does work in this case. Think about it.